### ISSUE TITLE
[LINUX] Dalamud plugin list download failed

### ISSUE THREAD

**Issue by ajiiisai** [goatcorp/FFXIVQuickLauncher#890]
### Update disclaimer

- [X] Yes, I have checked and my issue is not related to FFXIV updating and plugins not working correctly.

### What did you do?

It happens every time XIVLauncher updates to a new version.
![image](https://user-images.githubusercontent.com/85121274/163672028-ab390d5a-efcd-46e8-8070-962ff71b987b.png)

### Platform

Linux (Wine/Proton)

### Wine/Proton runner version

Lutris 7.2

**Comment by goaaats** [comment]
This happens when you switch Wine versions - you need to recreate your wine prefix for it to work again.

It's a Wine bug, and we can't really fix it, sorry. We'll have a better solution for Wine management soon.

**Comment by ajiiisai** [comment]
The thing is that I didn't switch Wine versions, I'm aware that there is a bug when you change it but that's not the case, I just updated the launcher from `6.2.28` to `6.2.29` an it happened.

**Comment by reiichi001** [comment]
Someone else modified the Lutris script without my knowledge and set it to Lutris-Wine-7.2-fshack. That version is broken for us. 

I've reverted it back to 7.1. 

Can you please remake your prefix with that version and test? You can pull your copy of FFXIV out along with your settings and pluginConfigs folder. See our FAQ for migration tips.

**Comment by ajiiisai** [comment]
Yeah remaking the prefix and symlinking the game worked fine. So it was probably the Lutris script.

**Comment by michaelagard** [comment]
> Yeah remaking the prefix and symlinking the game worked fine. So it was probably the Lutris script.

This isn't a tech support forum, but could you direct me to a guide or let me know how I can symlink the game to a new prefix?

**Comment by ajiiisai** [comment]
Just follow this: https://goatcorp.github.io/faq/xl_troubleshooting.html#q-how-do-i-migrate-ffxiv-andor-xivlauncher-files-from-an-old-wine-prefix-to-a-new-one-linux
